notebooks/2_feature_engineering.ipynb

```diff
@@ -189,77 +203,134 @@
     "* Running `fit_transform` on the documents return the embedding of each document. \n",
     "* Checkout different word options, pre-trained models, and other transformations.\n",
     "* See the rich list of [TextWiser Embeddings](https://github.com/fidelity/textwiser#available-embeddings)."
-   ]
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "# Sequential Data via Seq2Pat\n",
     "\n",
-    "* Time horizon is another source of information to build advanced recommenders.\n",
+    "* For most applications, structured data for user features via Selective and unstructured data for item features via TextWiser will suffice.\n",
+    "* In applications where events occur in sequences, time horizon can serve as another source of information to build advanced recommenders.\n",
+    "* We suggest starting simple and consider such sequential data to improve upon initial results once an end-to-end benchmarking is completed.\n",
     "* In recommenders, the typical usage of sequential data is to capture the behaviour of a user over time as part of **user features**. \n",
     "* For sequential data, you can leverage [Seq2Pat](https://github.com/fidelity/seq2pat).\n",
     "* Seq2Pat ([AAAI'22](https://aaai.org/Conferences/AAAI-22/)) provides an easy-to-use API for frequent pattern mining in sequential datasets. \n",
-    "* First, we find frequent patterns. Then, each user is transformed into a one-hot vector denoting the existence of frequent patterns in their sequential behaviour. This representation can be used as their user features.\n",
-    "* Let's explore a quick start example. "
-   ]
+    "* We first find frequent patterns with desired properties.\n",
+    "* Then, we represent each user with a one-hot vector denoting the existence of frequent patterns in user's sequential behaviour. This one-hot features can be used to augment user representation.\n",
+    "* Let's install the library and explore a quick start example."
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
   },
   {
    "cell_type": "code",
    "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# Install Seq2Pat\n",
+    "!pip install -q seq2pat;"
+   ],
    "metadata": {
+    "collapsed": false,
     "pycharm": {
      "name": "#%%\n"
     }
-   },
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
    "outputs": [],
    "source": [
     "# Example to show how to find frequent sequential patterns\n",
     "# from a given sequence database subject to constraints\n",
+    "# and generate one-hot encodings as the features of each user\n",
     "from sequential.seq2pat import Seq2Pat, Attribute\n",
+    "from sequential.dpm import get_one_hot_encodings\n",
```

Comment:
This is not about this PR, but external observation. Our API kinda looks weird now, no? 

Clearly, the user needs to employ get_encodings() when using seq2pat, not only when dealing with DPM. But get_encodings is defined in DPM. So even when you are not using DPM and all you want is seq2pat and its features, you need to "know about" DPM. Not cool, no?

Reply:
Perhaps what is worth doing is to move the encoding parts into another `featurization.py` script, then API becomes `from sequential.featurization import get_one_hot_encodings`? I suppose if there are other methods for featurization that we want to add in feature, e.g. embeddings from LSTM, those perhaps can go to featurization script.